### PR TITLE
add group management methods

### PR DESCRIPTION
**Pull request by luotuo29**
对组管理合约调用方法进行了封装

### CODE REVIEW

**Comment by luotuo29** [comment]
try to fix #125

**Comment by rainchen** [comment]
@luotuo29 

```
This branch has conflicts that must be resolved
```
please update and rebase your branch

**Comment by rainchen** [comment]
@luotuo29 please use `git rebase` to rebase your branch and use `git push -f` to update your branch, not expecting to see a new commit like "merge cita develop branch"

File: tests/src/main/java/com/cryptape/cita/tests/GroupManagerExample.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    public static void main(String[] args) throws Exception {
+        CITAjSystemContract sysContract = new CITAjSystemContract(service);
+
+        // New group
+        List<String> addresses = new ArrayList<>();
+        addresses.add("0x1c6eebf136ee234caff3a95e0d9d22e40c9ac4ca");
+        String newGroupAddr = sysContract.newGroup("vlk1", addresses, adminPrivateKey, version, chainId);
+        if ("".equals(newGroupAddr)) {
+            System.out.println("创建新group失败！");
```
Comment (rainchen) [inline tests/src/main/java/com/cryptape/cita/tests/GroupManagerExample.java]: should use English message

**Review by UDLD — COMMENTED** [review]
Don't delete config.properties.example. This file is one config set demo.

**Comment by luotuo29** [comment]
I have found that someone also tried to add functions about group management, and the function name were the same. So, l closed this pr.

**Pull request closed without merging**